plugins/mcreator-core/procedures/entity_create.json

```diff
@@ -0,0 +1,29 @@
+{
+  "args0": [
+    {
+      "type": "field_image",
+      "src": "./res/null.png",
+      "width": 8,
+      "height": 24
+    },
+    {
+      "type": "field_data_list_selector",
+      "name": "entity",
+      "datalist": "entity"
+    }
+  ],
+  "output": "Entity",
+  "colour": 195,
+  "mcreator": {
+    "toolbox_id": "advanced",
```

Comment:
I would put this under "Minecraft components"

Reply:
This is not some basic thing like dependency shortcut blocks (`coord_*`, `entity_from_deps`, etc.) or iterator blocks, it's a little hack to create a new entity but for "private use" and not for spawning.

Reply:
But is is Minecraft-related component, null entity is also in this category so this one fits there better

Reply:
**[null] No entity** is in the **Advanced** tab.

Reply:
Ok fair point, then it is indeed an appropriate location following what we did in the past for null entity. Sorry for the confusion, had in mind it was in mc components